tools/process-event/src/main.rs

```diff
@@ -83,6 +85,7 @@ impl Cli {
         }
 
         if self.store {
+            light_normalize_event(&mut event, &LightNormalizationConfig::default()).unwrap();
```

Comment:
On the other hand, do you really want to just `unwrap` it here ?  Maybe do same like other errors, add context and use `?` to bubble up the error?

Reply:
Good call, will map the error as the `process_value` calls do.